Kernel/Arch/aarch64/Registers.h

```diff
@@ -14,22 +14,25 @@ namespace Kernel::Aarch64 {
 
 // https://developer.arm.com/documentation/ddi0595/2021-06/AArch64-Registers/ID-AA64MMFR0-EL1--AArch64-Memory-Model-Feature-Register-0
 // Memory Model Feature Register 0
-struct ID_AA64MMFR0_EL1 {
-    int PARange : 4;
-    int ASIDBits : 4;
-    int BigEnd : 4;
-    int SNSMem : 4;
-    int BigEndEL0 : 4;
-    int TGran16 : 4;
-    int TGran64 : 4;
-    int TGran4 : 4;
-    int TGran16_2 : 4;
-    int TGran64_2 : 4;
-    int TGran4_2 : 4;
-    int ExS : 4;
-    int RES0 : 8;
-    int FGT : 4;
-    int ECV : 4;
+union ID_AA64MMFR0_EL1 {
```

Comment:
This does indeed also work, and apparently also forces the struct to be passed in a register. Since (for now at least) I don't need access to both the fields and the raw value, a struct+union is not necessary indeed. Will convert to `alignas(u64)`.